src/lib/datamodel/ClusterServer.h

```diff
@@ -76,7 +76,10 @@ class ClusterServer
      */
     Endpoint * GetEndpoint(uint8_t endpointId)
     {
-        return mEndpoints.Find([&endPointId](Endpoint * item) -> bool { return (endPointId-- == 0); });
+        /* Endpoint #0 is reserved, until we handle that properly, we will offset everything by 1 */
+        endpointId--;
+        auto *p = mEndpoints.Find([&endpointId](Endpoint * item) -> bool { return (endpointId-- == 0); });
```

Comment:
I don't understand this change.  endpoint 0 doesn't exist or is used for non-cluster commands?

also: less code to stop at `endpointId-- == 1` ?

Reply:
So as I understand, endpoint 0 is reserved and exists only if the device exposes a certain cluster. So skipping it for now.

Reply:
done